Automattic/newspack-theme
# Building out base styles and structure for single articles

### All Submissions:


### Changes proposed in this Pull Request:

This PR makes updates to the single article's appearance and order to match the [mockups](https://cloudup.com/irxJe2y49aX); specific changes include:

* Moving Jetpack's sharing icon output to site up next to the title.
* Moving the category output above the title.
* Adjusts general styles and spacing. 

### How to test the changes in this Pull Request:

1. Apply the PR. 
2. Activate Jetpack's share buttons; set to just show icons.
3. Review single stories and compare against the [mockups](https://cloudup.com/irxJe2y49aX); make sure they generally look correct. Note: the sharing icons display different with AMP enabled (large and rectangular, not smaller and rounded) - I'm less familiar with the styling there, so have opted not to try to override it for now. 

### Other information:
